config/federationconfig.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: core.federation.k8s.io/v1alpha1
+kind: FederationConfig
+metadata:
+  name: federation-v2
+  namespace: federation-system
+spec:
+  limited-scope: false
+  registry-namespace: kube-multicluster-public
+  controller-duration:
+    available-delay: 20s
+    unavailable-delay: 60s
+    cluster-monitor-period: 40s
+  leader-elect:
+    lease-duration: 1500ms
```

Comment:
Given that these are test-specific values, I'd like to avoid putting this in a location where a user might assume it was something that should be used for non-test purposes.   I also don't think it's necessary to supply non-default values for everything.  That said, I'll fix in a follow-up PR rather than delaying this one.